steps/update-jira-release/0.0.1/step.yml

```diff
@@ -0,0 +1,89 @@
+title: Jira Release Status Update
```

Comment:
Would recommend something like: `Update Jira Release Status` (https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title)